README.md

```diff
@@ -68,6 +68,23 @@ admin_cidr = "0.0.0.0/0"
 
 For more advanced scenarios, please check the [terraform-dcos documentation for Google Cloud](https://github.com/dcos/terraform-dcos/tree/master/gcp).
 
+### Kubernetes configuration
+
+**NOTE:** By default, it will provision a Kubernetes cluster with one (1) worker node, and
+a single instance of every control plane component.
+
+To deploy a **highly-available** cluster with three (3) private and one (1) public workers node update `.deploy/options.json`:
```

Comment:
Shame we can't just use resources/option-ha.json somehow?

Reply:
HA options aren't same per cloud provider

Reply:
The only difference is for AWS, right? Anyway, we can maybe improve that in future, this is great for now.